nix/apps.nix

```diff
@@ -6,6 +6,12 @@
     apps = let
       inherit (self'.packages) sbomnix;
     in {
+      # nix run .#sbomnix
```

Comment:
Don't we already have this in the default output? cc @brianmcgee

Reply:
Yeah `sbomnix` is the default package and is also a 'named' package. It doesn't require an app @henrirosten .

It can be run with either `nix run .# -- --help` or `nix run .#sbomnix -- --help`

Reply:
Fixed:  https://github.com/tiiuae/sbomnix/compare/2a1f91600695b7261b70eb3f82ca4c437d5039ac..74f874012559a239f64f6af57d564e0869ee1554